src/assets/less/common.less

```diff
@@ -200,11 +236,37 @@ img {
   margin-top: -82px;
 }
 
-.card-body {
-  padding: 0;
+div.card-body {
+  padding: 0px;
```

Comment:
Did you check how this interacts with the riven pages, since these accordions sometimes include card body?

Reply:
Yes, Riven page UI is still the same.